### DISCUSSION TITLE
lightdm keeps failing constantly

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#219] [Q&A]
in my LBB, I see lightdm constantly failing

from /var/log/lightdm/lightdm.log:

`[+889.83s] DEBUG: XServer 0: Logging to /var/log/lightdm/x-0.log
[+889.83s] DEBUG: XServer 0: Writing X server authority to /var/run/lightdm/root/:0
[+889.83s] DEBUG: XServer 0: Launching X Server
[+889.83s] DEBUG: Launching process 29481: /usr/bin/X :0 -seat seat0 -auth /var/run/lightdm/root/:0 -nolisten tcp vt7 -novtswitch
[+889.84s] DEBUG: XServer 0: Waiting for ready signal from X server :0
[+889.87s] DEBUG: Process 29481 exited with return value 1
[+889.87s] DEBUG: XServer 0: X server stopped
[+889.87s] DEBUG: XServer 0: Removing X server authority /var/run/lightdm/root/:0
[+889.87s] DEBUG: Seat seat0: Display server stopped
[+889.87s] DEBUG: Seat seat0: Failed to start display server - trying to restart
[+889.87s] DEBUG: XServer 0: Logging to /var/log/lightdm/x-0.log
[+889.87s] DEBUG: XServer 0: Writing X server authority to /var/run/lightdm/root/:0
[+889.87s] DEBUG: XServer 0: Launching X Server
[+889.88s] DEBUG: Launching process 29482: /usr/bin/X :0 -seat seat0 -auth /var/run/lightdm/root/:0 -nolisten tcp vt7 -novtswitch
[+889.88s] DEBUG: XServer 0: Waiting for ready signal from X server :0
[+889.91s] DEBUG: Process 29482 exited with return value 1
[+889.91s] DEBUG: XServer 0: X server stopped
[+889.91s] DEBUG: XServer 0: Removing X server authority /var/run/lightdm/root/:0
[+889.91s] DEBUG: Seat seat0: Display server stopped
[+889.91s] DEBUG: Seat seat0: Failed to start display server - trying to restart
[+889.91s] DEBUG: XServer 0: Logging to /var/log/lightdm/x-0.log
[+889.92s] DEBUG: XServer 0: Writing X server authority to /var/run/lightdm/root/:0
[+889.92s] DEBUG: XServer 0: Launching X Server
[+889.92s] DEBUG: Launching process 29483: /usr/bin/X :0 -seat seat0 -auth /var/run/lightdm/root/:0 -nolisten tcp vt7 -novtswitch
[+889.92s] DEBUG: XServer 0: Waiting for ready signal from X server :0
[+889.96s] DEBUG: Process 29483 exited with return value 1
[+889.96s] DEBUG: XServer 0: X server stopped
[+889.96s] DEBUG: XServer 0: Removing X server authority /var/run/lightdm/root/:0
[+889.96s] DEBUG: Seat seat0: Display server stopped
[+889.96s] DEBUG: Seat seat0: Failed to start display server - trying to restart
[+889.96s] DEBUG: XServer 0: Logging to /var/log/lightdm/x-0.log
[+889.96s] DEBUG: XServer 0: Writing X server authority to /var/run/lightdm/root/:0
[+889.96s] DEBUG: XServer 0: Launching X Server
[+889.96s] DEBUG: Launching process 29484: /usr/bin/X :0 -seat seat0 -auth /var/run/lightdm/root/:0 -nolisten tcp vt7 -novtswitch
[+889.96s] DEBUG: XServer 0: Waiting for ready signal from X server :0
[+890.00s] DEBUG: Process 29484 exited with return value 1
[+890.00s] DEBUG: XServer 0: X server stopped
[+890.00s] DEBUG: XServer 0: Removing X server authority /var/run/lightdm/root/:0
[+890.00s] DEBUG: Seat seat0: Display server stopped
[+890.00s] DEBUG: Seat seat0: Failed to start display server - trying to restart
[+890.00s] DEBUG: XServer 0: Logging to /var/log/lightdm/x-0.log
[+890.00s] DEBUG: XServer 0: Writing X server authority to /var/run/lightdm/root/:0
[+890.00s] DEBUG: XServer 0: Launching X Server
[+890.01s] DEBUG: Launching process 29485: /usr/bin/X :0 -seat seat0 -auth /var/run/lightdm/root/:0 -nolisten tcp vt7 -novtswitch
[+890.01s] DEBUG: XServer 0: Waiting for ready signal from X server :0
[+890.05s] DEBUG: Process 29485 exited with return value 1
[+890.05s] DEBUG: XServer 0: X server stopped
[+890.05s] DEBUG: XServer 0: Removing X server authority /var/run/lightdm/root/:0
[+890.05s] DEBUG: Seat seat0: Display server stopped
[+890.05s] DEBUG: Seat seat0: Failed to start display server - trying to restart
[+890.05s] DEBUG: XServer 0: Logging to /var/log/lightdm/x-0.log
[+890.05s] DEBUG: XServer 0: Writing X server authority to /var/run/lightdm/root/:0
[+890.05s] DEBUG: XServer 0: Launching X Server`

The x-0.log Xserver log  gets quickly overwritten, but does not seem to have anything useful when i cought it)

**Comment by outdoorbits** [comment]
Sorry, can't help, I'm working on headless RPi only.
